x-pack/plugins/endpoint/public/applications/endpoint/index.tsx

```diff
@@ -52,6 +66,101 @@ interface RouterProps {
   store: Store;
   coreStart: CoreStart;
   depsStart: EndpointPluginStartDependencies;
+  ingestManager: IngestManagerSetup;
+}
+
+const IsAppUnavailable: React.FunctionComponent<{
```

Comment:
(optional) From an organization standpoint, maybe move this component to `views`?

Also - don't we (endpoint) have do do some setup after ingest, triggered from the UI? or is that triggered on the API side?

If we also need to triger setup, then it might be a good ideal to move this to its own component (maybe called `Setup`) and have it handle all that business 😃

Reply:
> (optional) From an organization standpoint, maybe move this component to `views`?

Good idea, I'll move it to `views`

> 
> Also - don't we (endpoint) have do do some setup after ingest, triggered from the UI? or is that triggered on the API side?

I don't think there's any other setup that we'll have to do. The ingest manager handles installing the package for us and everything.

> 
> If we also need to triger setup, then it might be a good ideal to move this to its own component (maybe called `Setup`) and have it handle all that business 😃